templates/express_mongo_ts/src/routes/index.ts

```diff
@@ -0,0 +1,14 @@
+import { Router, Request, Response } from "express";
+import { user } from "./user.routes";
```

Comment:
The `index.ts` file inside `routes/` is a central location where we are aggregating all the different routes which will be created by the user to handle different endpoints that will be defined in different route-files under the same directory. 

This will remain unchanged.